.github/workflows/labeler.yml

```diff
@@ -1,15 +1,15 @@
 name: "Pull Request Labeler"
 
 on:
-  schedule:
-  - cron: "0 0 */1 * *"
```

Comment:
You told me to switch to a corn job because the approach you're proposing now is unable to run on PRs coming from forks. This is because the action is running inside the fork, but needs to access this repo (the upstream). That's why you get the error.

Reply:
OK, now I remember.
I will try to fix this.

Reply:
> You told me to switch to a corn job

Please keep that to yourself, this is not cornhub.com

Reply:
> this is not cornhub.com

yet